setup.py

```diff
@@ -36,6 +40,11 @@
     packages=['sanic'],
     platforms='any',
     install_requires=install_requires,
+    extras_require={
+        "all": install_requires + ujson + uvloop,
+        "ujson": install_requires + ujson,
+        "uvloop": install_requires + uvloop,
```

Comment:
Is `install_requires` really supposed to be here? Shouldn't this just contain the *extra* packages, i.e. whatever package besides what's already defined in `install_requires`.